src/client/App.tsx

```diff
@@ -303,6 +304,9 @@ function AppAuthenticated() {
               component={ToeristscheVerhuurThema}
             />
           )}
+          {FeatureToggle.varenActive && (
+            <Route path={AppRoutes['VAREN']} component={Varen} />
```

Comment:
Deze syntax kan simpler.
```suggestion
            <Route path={AppRoutes.VAREN} component={Varen} />
```